crates/shared/src/types.rs

```diff
@@ -105,6 +105,8 @@ pub enum HashingAlgorithm {
     Sha3,
     Keccak,
     Blake2_256,
+    /// If you are trying to sign a hashed message you can use this, length requirement is 64 digits
```

Comment:
```suggestion
    /// An algorithm which produces the same output as its input.
```

Not sure we should be having such specific "advice" or use cases as part of the doc comments. All the hashers output 32 bytes (256-bits) here as well.